server/src/utils/mixerConnections/VMixMixerConnection.ts

```diff
@@ -166,6 +166,15 @@ export class VMixMixerConnection {
                     ) // add +15 to convert from dBFS
                 }
 
+                // If vMix has more channels than Sisyfos is configured to handle,
+                // then do nothing with those additional channels.
+                if (
+                    !state.channels[0].chMixerConnection[this.mixerIndex]

```

Comment:
This is not the most clear conditional to read. Could be nice to extract so it atleast gets a nice name:
```
if (!doesChannelExists(...)) {
  return
}

...

function doesChannelExists(...): boolean {
  return !!state.channels[0].chMixerConnection[this.mixerIndex].channel[input.number - 1]
}
```